crates/bevy_ui/src/render/mod.rs

```diff
@@ -440,7 +451,7 @@ pub fn extract_uinodes(
             ExtractedUiNode {
                 stack_index: uinode.stack_index,
                 transform: transform.compute_matrix(),
-                color: color.0,
+                color: opacity * color.0,
```

Comment:
This can get very strange if opacity is > 1 or < 0, which can happen very easily, we might want to clamp somewhere on the CalculatedOpacity and warn the user to use only opacity between 0 and 1 maybe ?

Reply:
Is this fix sufficient? We could also clamp the Opacity during propagation.  https://github.com/bevyengine/bevy/blob/55368a1446e5e72018a6b387f0be1eb32498734d/crates/bevy_ui/src/ui_node.rs#L1666-L1674

Reply:
I think this is good for now, another possibility would be clamp in the calc function, for example
Given the parent Opacity is 0.5, and the children opacity is 2, all the children of the given children would have a bigger opacity than the root parent, if we only clamp there.